process/smartContract/process_test.go

```diff
@@ -539,13 +546,20 @@ func TestScProcessor_DeploySmartContract(t *testing.T) {
 	tx.SndAddr = []byte("SRC")
 	tx.RcvAddr = generateEmptyByteSlice(addrConverter.AddressLen())
 	tx.Data = "data"
-	tx.Value = big.NewInt(45)
+	tx.Value = big.NewInt(0)
```

Comment:
Should revert to 45 as 0 is a neutral element when adding or subtracting.

Reply:
discussed.